packages/template/android/src/main/cpp/cpp-adapter.cpp

```diff
@@ -2,15 +2,13 @@
 
 #include "JFunc_void_Person.hpp"
 #include "JFunc_void_std__string.hpp"
-#include "JHybridImageFactorySpec.hpp"
-#include "JHybridImageSpec.hpp"
 #include "JHybridKotlinTestObjectSpec.hpp"
 
 #include "HybridTestObject.hpp"
 #include <NitroModules/HybridObjectRegistry.hpp>
 #include <fbjni/fbjni.h>
 
-using namespace margelo::nitro::image;
+using namespace margelo::nitro::<<androidCxxLibName>>;
```

Comment:
One solution could also be to not go this deep in the namespace in the template, like we have on iOS. Then there is no need for this

Reply:
This then could cause nameclashes :)